os/board/rtl8730e/src/rtl8730e_mipi_lcdc.c

```diff
@@ -42,37 +45,36 @@
 #define MIPI_GPIO_RESET_PIN 	PA_14
 #endif
 
-struct rtl8730_lcdc_info_s {
+struct rtl8730e_lcdc_info_s {
```

Comment:
I am sorry but let's split code change (operation change) and code cleanup.

Reply:
done~